arches/app/tasks.py

```diff
@@ -12,6 +12,7 @@
 from arches.app.search.search_export import SearchResultsExporter
 import arches.app.utils.zip as zip_utils
 from django.utils.translation import ugettext as _
+from django.conf import settings
```

Comment:
Sorry, another comment.  I have seen in most other arches places that they have used the following to import the settings module...

```suggestion
from arches.app.models.system_settings import settings
```
Did you use this because it was a celery task?

Reply:
I did that because in some files it was talking about there being a conflict and recursive calling of the system_settings so I was playing things safe.  If they're generally using that in these app files though then let's follow their preferences unless it proves to be an issue.